### DISCUSSION TITLE
filament-tables: combine DB columns in cell?

### DISCUSSION THREAD

**Discussion by jrmypttrsn** [filamentphp/filament#2394] [Help] [answered]
In a previous table I had this column:
![status-old](https://user-images.githubusercontent.com/16291625/167988323-c1ca0e9e-4a2c-4033-9d6a-15c6ad1905c5.png)

Using filament-tables I have a view file where I style the certification status but I can't figure out how to get the date from another column in with it:
![status-filament](https://user-images.githubusercontent.com/16291625/167988400-9df47520-471e-456b-b8df-9a9f59c2d063.png)

Is it possible to combine DB columns in a cell?

**Comment by jrmypttrsn** [comment, marked as answer]
🤦🏻‍♂️ Just saw getRecord(). It's late...

**Reply by OvalMedia** [reply]
Would you mind sharing what you did with it?
